avocado/core/safeloader.py

```diff
@@ -264,59 +323,58 @@ def _examine_class(path, class_name):
                 # a module
                 continue
             parent_class = parent.id
-            _info, _disabled = _examine_class(path, parent_class)
+            _info, _disabled, _avocado = _examine_class(path, parent_class,
+                                                        is_avocado)
             if _info:
                 parents.remove(parent)
                 info.extend(_info)
                 disabled.update(_disabled)
+            if _avocado is not is_avocado:
```

Comment:
I know what you're up to here, but this amount of similarly named variables makes readability a bit tough.  Suggestion is to increase the verbosity of names if necessary to increase readability and maintainability.

Reply:
Well the `_` variables are looping/temporary variables. It does not look confusing to me. But if you have a better suggestion, I can refactor that.